keyvi/include/keyvi/dictionary/completion/prefix_completion.h

```diff
@@ -173,7 +173,7 @@ class PrefixCompletion final {
           if (data->traverser) {
             TRACE("Current depth %d", exact_prefix + data->traverser.GetDepth() - 1);
 
-            int score =
+            size_t score =
```

Comment:
a score isn't a size type. What's the problem with int? I am open for replacing it with int32_t, to make it more explicit as int isn't clearly defined. A 64bit type would be to large, given that for the matrix calculation it would mean a massive size increase.